web_monitoring/differs.py

```diff
@@ -5,3 +5,7 @@ def dummy(a, b, **options):
 
 def compare_length(a, b):
```

Comment:
Definitely. All this checks is the length. If the length is *different*, then we know that the byte-strings are different. This may or may not be useful; if nothing else it makes for a simple instructive example.